dvc/repo/experiments/queue/workspace.py

```diff
@@ -1,11 +1,13 @@
 import logging
+import os
 from collections import defaultdict
 from typing import TYPE_CHECKING, Collection, Dict, Generator, Optional
 
 from funcy import first
 
 from dvc.exceptions import DvcException
 
+from ....utils.fs import remove
```

Comment:
Please don't use these kinds of imports. 

```suggestion
from dvc.utils.fs import remove
```